config/global.js

```diff
@@ -82,6 +83,7 @@ const exp = {
 	SC_SUPERHERO_USER_PASSWORD,
 	SC_SUPERHERO_USER_NAME,
 	HELPAREA_URL,
+	FEATURE_NUXT_SCHOOL_ADMIN_BETA,
```

Comment:
default.schema.json are the better place but this is ok as temporary solution.

Reply:
Ah ok it is alread in default.schema.json. Than it should not nessasrary to add it to global too.